brain/validator/validate_password.py

```diff
@@ -44,8 +43,7 @@ def load_max(app=True, root='/vagrant'):
     else:
         with open(root + "/hiera/settings.yaml", 'r') as stream:
             try:
-                max_c = \
-                       yaml.load(stream)['validate_password']['password_max_c']
+                max_c =yaml.load(stream)['validate_password']['password_max_c']

```

Comment:
Oops, I read your previous comment wrong, didn't realize you said to split into yamlres, thought you were saying to get rid of the space